docs/tutorials/excel-tutorial.md

```diff
@@ -16,7 +16,7 @@ This tutorial will teach you the basics of recording, editing, and writing an Of
 Before starting this tutorial, you'll need access to Office Scripts, which requires the following:
 
 - Excel on the web.
-- [Opt-in to the Office Scripts preview (admin-enabled)](https://aka.ms/office-scripts-admin). This adds the **Automate** tab to the ribbon.
+- Have your [admin enable the Office Scripts for your account](https://support.office.com/article/office-scripting-settings-in-m365-19d3c51a-6ca2-40ab-978d-60fa49554dcf). This adds the **Automate** tab to the ribbon.
```

Comment:
Can we change the text to read:
Enable Office Scripts on your account, which adds the Automate tab to the ribbon
?

Reply:
I cleaned that line up a bit, During the docs bash, it was asked we make it very clear the admin needs to activate this feature, not the end-user.

Reply:
Got it. How about: 

Ask you administrator to enable Office Scripts on your account, which adds the Automate tab to the ribbon.

?